app/views/content/non-uk-teachers/visas-for-non-uk-trainees.md

```diff
@@ -130,14 +130,14 @@ You may be able to train part-time, but your salary must meet the threshold for
 
 There are 2 requirements:
 
-Your part-time salary must be at least £20,960.
+Your part-time salary must be at least £23,200.
 
 The full-time equivalent of your salary must meet the threshold set for the skilled worker visa in your region, as follows: 
 
-* inner London: £23,099
-* outer London: £21,832
-* London fringe: £20,960
-* rest of England: £20,960
+* inner London: £25,831
+* outer London: £24,415
+* London fringe: £23,200
+* rest of England: £23,200
```

Comment:
@laurahermionetennant92 same as above comments re if the amount should be the same for London fringe and rest of England.